app/templates/components/notification-dropdown.hbs

```diff
@@ -0,0 +1,59 @@
+    <div class="ui wide notification popup bottom lef transition ">
+  <div class="ui fluid link  divided list">
+    <div class="item">
+      <div class="right floated content">
+        <a>{{t 'Mark all as Read'}}</a>
+      </div>
+    </div>
+    <div class="item">
+      <div class="content">
+        <a class="header"><span>New Session Propoal</span></a>
+        <div class="description">{{t 'Event'}} : <a class="item"><b>Sample</b></a></div>
```

Comment:
The description will just be plain text. 

And also, don't use translation helpers for dummy text. Doesn't make any sense.